src/Fixer/Operator/PreIncrementFixer.php

```diff
@@ -29,19 +29,19 @@
     public function getDefinition()
     {
         return new FixerDefinition(
-            sprintf('Pre incrementation/decrementation should be used if possible. DEPRECATED: Use "%s" instead.', $this->proxyFixer->getName()),
+            'Pre incrementation/decrementation should be used if possible. DEPRECATED: Use "increment_style" instead.',
```

Comment:
why hardcoding the name ?

Reply:
Because I would have to hardcode it to retrieve the proxied fixer anyway, i.e. `$this->proxyFixers['increment_style']->getName()`.

Reply:
or `current($this->proxyFixers)->getName()`, how you just did in `HashToSlashCommentFixer`

Reply:
Done.